index.js

```diff
@@ -23,6 +23,6 @@ function fastifySwagger (fastify, opts, next) {
 }
 
 module.exports = fp(fastifySwagger, {
-  fastify: '>=0.39.0',
+  fastify: '>=1.12.1',
```

Comment:
I fear this might fail in v2, as it depends on fastify-static.

Reply:
Should we bump `fastify-static` to the same version as well?

Reply:
no, not really. fastify-static should be `^1.0.0` ideally.